taniumrest_connector.py

```diff
@@ -555,15 +560,15 @@ def _execute_action_support(self, param, action_result): # noqa: 901
                     "The following key(s) are incorrect: {}. Please provide correct key(s)".format(', '.join(invalid_keys)))
 
         data = dict()
-        package_param = dict()
+        package_param = list()
```

Comment:
```suggestion
        package_param = []
```

Reply:
we have used functions all over the code rather than comprehensions
Change will break the consistency

Reply:
Not writing unidiomatic, and for that matter less performant, though in this case this is purely pedantic point, code is IMHO more important than consistency

Reply:
agree that we should always strive for better code
As there is negligible difference in performance between both syntax, I prefer to keep the code consistent

Reply:
@bb-splunk Any inputs from your side?